Nitrox.Assets.Subnautica/LanguageFiles/en.json

```
@@ -65,5 +65,7 @@
     "Nitrox_WaitingPassword": "Waiting for server password input…",
     "Nitrox_WaitingUserInput": "Waiting for user input…",
     "Nitrox_WorldSettling": "Awaiting World Settling…",
-    "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target."
+    "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target.",
```

I might be wrong but a `'s` after a word that isn't `it` will be read as possessive instead of a contraction of `is`.

```suggestion
    "Nitrox_RemotePlayerObstacle": "Another player is either inside or too close to the deconstructable target.",
```

---

I didn't change that, I can try it out to see if it works ? I just added a comma on the end of the line

---

Right, it's out of scope of this PR..